src/Raven.Server/NotificationCenter/Notifications/Details/LargetToolResponseDetails.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.NotificationCenter.Notifications.Details
+{
+    public sealed class ExceededTokenThresholdDetails : INotificationDetails
+    {
+        private ExceededTokenThresholdDetails() { }
+
+        private ExceededTokenThresholdDetails(string agentName, string conversationId, int tokenCount, int tokenLimit, List<ToolCallDetails> toolCalls, string recommendation)
+        {
+            AgentName = agentName;
+            ConversationId = conversationId;
+            TokenCount = tokenCount;
+            TokenThreshold = tokenLimit;
+            ToolCalls = toolCalls;
+            Recommendation = recommendation;
+        }
+
+        public string AgentName { get; set; }
+        public string ConversationId { get; set; }
+        public int TokenCount { get; set; }
+        public int TokenThreshold { get; set; }
+        public List<ToolCallDetails> ToolCalls { get; set; }
+        public string Recommendation { get; set; }
+
+        public DynamicJsonValue ToJson()
+        {
+            return new DynamicJsonValue(GetType())
+            {
+                [nameof(AgentName)] = AgentName,
+                [nameof(ConversationId)] = ConversationId,
+                [nameof(TokenCount)] = TokenCount,
+                [nameof(TokenThreshold)] = TokenThreshold,
+                [nameof(ToolCalls)] = new DynamicJsonArray(ToolCalls.Select(tc => tc.ToJson())),
+                [nameof(Recommendation)] = Recommendation
+            };
+        }
+
+        public static void Add(DatabaseNotificationCenter notificationCenter, string agentName, string conversationId, int tokenCount, int tokenThreshold, List<ToolCallDetails> toolCalls)
+        {
+            var message = $"In conversation '{conversationId}', the AI Agent '{agentName}' sent a request with {tokenCount} tokens to the LLM, exceeding the threshold of {tokenThreshold} tokens.";
```

Comment:
threshold => configured threshold

also I would add using which configuration key this can be changed? use Raven.Configuration.GetKey method to get they key